sql/moz-fx-data-shared-prod/monitoring_derived/metadata_completeness_v1/query.sql

```diff
@@ -0,0 +1,91 @@
+--Get all objects in moz-fx-data-shared-prod or mozdata
+WITH all_mozdata_and_shared_prod_objects AS (
+  SELECT
+    table_catalog,
+    table_schema,
+    table_name,
+    table_type
+  FROM
+    `moz-fx-data-shared-prod.region-US.INFORMATION_SCHEMA.TABLES`
+  UNION ALL
+  SELECT
+    table_catalog,
+    table_schema,
+    table_name,
+    table_type
+  FROM
+    `mozdata.region-US.INFORMATION_SCHEMA.TABLES`
+  WHERE
+    LOWER(table_schema) <> 'looker_tmp'
+),
+--get the object descriptions if there are any
+mozdata_and_shared_prod_object_descriptions AS (
+  SELECT
+    table_catalog,
+    table_schema,
+    table_name,
+    option_value AS object_description
+  FROM
+    `moz-fx-data-shared-prod.region-US.INFORMATION_SCHEMA.TABLE_OPTIONS`
+  WHERE
+    option_name = 'description'
+  UNION ALL
+  SELECT
+    table_catalog,
+    table_schema,
+    table_name,
+    option_value AS object_description
+  FROM
+    `mozdata.region-US.INFORMATION_SCHEMA.TABLE_OPTIONS`
+  WHERE
+    option_name = 'description'
+),
+--get the # of columns on each object and how many have a description
+shared_prod_and_mozdata_column_metadata_completeness AS (
+  SELECT
+    table_catalog,
+    table_schema,
+    table_name,
+    COUNT(1) AS nbr_columns,
+    SUM(CASE WHEN description IS NOT NULL THEN 1 ELSE 0 END) AS nbr_columns_with_non_null_desc
```

Comment:
Just a note that I'd normally use COUNT to count non nulls, since that's what COUNT does.